test/gui/tst_syncing/test.feature

```diff
@@ -480,3 +480,21 @@ Feature: Syncing files
         Then the folder "simple-folder" should not exist on the file system
         And the folder "test-folder/sub-folder2" should exist on the file system
         And the folder "test-folder/sub-folder1" should not exist on the file system
+
+
+    Scenario: Add folder to sync
```

Comment:
Maybe we can integrate these to existing scenario (add folder sync connection) if possible